_posts/2021-01-12-updating-foreman-inventory-with-system-facts.md

```diff
@@ -0,0 +1,323 @@
+    </tr>
+    <tr>
+      <td>Ignore facts for domain</td>
+      <td>Provisioning</td>
+      <td>By default, Foreman will be creating and assigning Domain objects based on the host domain name. If you don't want this to be updated, you can disable this behavior using this setting.</td>
+    </tr>
+    <tr>
+      <td>Ignore facts for operating system</td>
+      <td>Provisioning</td>
+      <td>By default, Foreman will be creating and Operating System objects if they do not exist in Foreman yet. Also the host will be associated to such OS (either found or newly created), which is useful when OS is upgraded on the host without Foreman knowing about it. In case you e.g. upgrade RHEL 7.2 to 7.3 and there's no RHEL 7.3 on your Foreman yet, it will be created. Note that newly create OS is not fully configured for provisioning. E.g. there are no Provisioning Templates assigned to it, since we don't know which should be used. Therefore later, when you decide to reprovision the host, you may be surprised you need to provide additional configuration for the Operating System first</td>
+    </tr>
+    <tr>
+      <td>Ignore interfaces with matching identifier</td>
+      <td>Provisioning</td>
+      <td>This is similar to excluded pattern settings above, it's only limited to parsing network interfaces facts. Parsing NICs is more performance heavy operation, therefore has its own configuration. But in general, it can be superceeded by the mentioned setting.</td>
+    </tr>
+    <tr>
+      <td>Ignore Puppet facts for provisioning</td>
+      <td>Provisioning</td>
+      <td>Completely ignore networking information importing from facts, this is useful when the detection does not correctly work in your environment, e.g. you use custom names for bonds or vlan interfaces.</td>
+    </tr>
+  </tbody>
+</table>
+
+### Configuring the Ansible callback
+
+If you're an Ansible user, there is an easy way to keep your Foreman inventory up to date thanks to the foreman.py callback.
+It is also an elegant way to populate (import) unmanaged hosts to your Foreman instance. If you use the installer and install
+[foreman_ansible plugin](https://theforeman.org/plugins/foreman_ansible/), the callback is deployed for any Ansible run
+triggered from Foreman.
+
+However we'll look at how to configure the callback manually into your existing Ansible directory. That way, if you run
+any Ansible playbook, Ansible will upload facts and report to your Foreman. If one or more hosts that the Ansible run
+included does not exist in your Foreman instance, it will be created. If it exists, it will be updated based on the facts
+gathered during the Ansible run. You still need the foreman_ansible to be installed to your Foreman, so it can properly import and process Ansible facts.
```

Comment:
```suggestion
gathered during the Ansible run. You still need the foreman_ansible plugin to be installed to your Foreman, so it can properly import and process Ansible facts.
```

Reply:
Maybe add a link here to the foreman_ansible install guide?